recipe/meta.yaml

```
@@ -26,7 +26,7 @@ requirements:
     - graphene >=2.1,<3
     - jinja2 ==2.11.0,<2.12
     - metomi-isodatetime ==1!2.0.1
-    - protobuf ==3.12.1
+    - protobuf =>3.15.0,<3.16.0
```

```suggestion
    - protobuf >=3.15.0,<3.16.0
```

Oops, and we all approved that :rofl: